### PR TITLE
Resolved conflict

### PR DESCRIPTION
**Pull request by CleverChuk**
* Resolved conflict with Emmanuels commit
* [ Data binding doc](https://developer.android.com/topic/libraries/data-binding/index.html)

### CODE REVIEW

**Comment by Madonahs** [comment]
Awesome, let me check it Out. Well done team work :) 💯 👍

File: app/src/main/java/com/madonasyombua/growwithgoogleteamproject/ui/fragment/RegisterFragment.java [hunk 1]
```diff
@@ -54,18 +43,6 @@ public void onClick(View view) {
                 String username_txt = extractText(binding.editUsername);
```
Comment (MaherSoua) [inline app/src/main/java/com/madonasyombua/growwithgoogleteamproject/ui/fragment/RegisterFragment.java]: Please use naming convention , we should use camelCase in our case if the variable is member we name like that mVariable

Comment (Madonahs) [reply app/src/main/java/com/madonasyombua/growwithgoogleteamproject/ui/fragment/RegisterFragment.java]: Yeah, like mMaster :) Agreed. So far so good project.

**Comment by Madonahs** [comment]
@CleverChuk  @MaherSoua @emmaunel Well done Team. Master is good to go. 🥇

**Comment by Madonahs** [comment]
**Encountering Data binding on lesson 12 : 24. Finally.**

**Pull request merged**